packages/odyssey-react-mui/src/Grid.tsx

```diff
@@ -0,0 +1,134 @@
+   */
+  md?: MuiGridProps["md"];
+  /**
+   * If a number, it sets the number of columns the grid item uses. It can't be greater than the total number of columns of the container (12 by default). If 'auto', the grid item's width matches its content. If false, the prop is ignored. If true, the grid item's width grows to use the space available in the grid container. The value is applied for the `sm` breakpoint and wider screens if not overridden.
+   */
+  sm?: MuiGridProps["sm"];
+  /**
+   * If a number, it sets the number of columns the grid item uses. It can't be greater than the total number of columns of the container (12 by default). If 'auto', the grid item's width matches its content. If false, the prop is ignored. If true, the grid item's width grows to use the space available in the grid container. The value is applied for all the screen sizes with the lowest priority.
+   */
+  xs?: MuiGridProps["xs"];
+};
+
+export type GridProps = {
+  /**
+   * The `Grid` content. Will be made up of `<Grid item />` components
+   */
+  children: MuiGridProps["children"];
+  /**
+   * The number of columns.
+   */
+  columns?: MuiGridProps["columns"];
+  /**
+   * The component used for the root node. Either a string to use a HTML element or a component.
+   */
+  component?: MuiGridProps["component"];
+  /**
+   * If true, the component will have the flex container behavior. You should be wrapping items with a container.
+   */
+  container?: MuiGridProps["container"];
+  /**
+   * Defines the horizontal space between the type item components. It overrides the value of the spacing prop.
+   */
+  columnSpacing?: MuiGridProps["columnSpacing"];
+  /**
+   * Defines the flex-direction style property. It is applied for all screen sizes.
+   */
+  direction?: MuiGridProps["direction"];
+  /**
+   * designates the `Grid` component as a child item of it's parent container
+   */
+  item?: MuiGridProps["item"];
+  /**
+   * Defines the vertical space between the type item components. It overrides the value of the spacing prop.
+   */
+  rowSpacing?: MuiGridProps["rowSpacing"];
+  /**
+   * Defines the space between the type item components. It can only be used on a type container component.
+   */
+  spacing?: MuiGridProps["spacing"];
+};
+
+export type GridContainerProps = Pick<
+  GridProps,
+  | "children"
+  | "component"
+  | "columnSpacing"
+  | "direction"
+  | "rowSpacing"
+  | "spacing"
+>;
+
+export type GridItemProps = Pick<GridProps, "children" | "component"> &
+  GridBreakpoints;
+
+const Grid = ({
+  children,
+  columns,
+  component = "div",
+  container,
+  direction,
+  item,
+  rowSpacing,
+  spacing,
+}: GridProps) => {
+  return (
+    <MuiGrid
+      columns={columns}
+      container={container}
+      component={component}
+      direction={direction}
```

Comment:
Do we have a Storybook entry for `Grid`? I'm not sure I completely understand what it does without seeing it.

Reply:
I removed Grid from this PR